packages/frameworks/src/frameworks.ts

```diff
@@ -849,7 +813,6 @@ export const frameworks = [
       },
     },
     dependency: '@scullyio/init',
-    getFsOutputDir: async () => 'dist',
```

Comment:
This one has a different value than `getOutputDirName()`. Maybe this one needs to stay?

Reply:
@jaredpalmer Do you know why this one was different?

Reply:
Looks like it's because `dist/static` exists, so it already _sorta_ conforms to the FS API if we consider `dist` as the output dir.